src/Core/Models/SqlTypeConstants.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Service.Models;
+
+/// <summary>
+/// String literal representation of SQL Server data types that are returned by Microsoft.Data.SqlClient.
+/// </summary>
+/// <seealso cref="https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/tests/PerformanceTests/Config/Constants.cs"/>
+public static class SqlTypeConstants
+{
+    /// <summary>
+    /// SqlDbType names ordered by corresponding SqlDbType.
+    /// Keys are lower case to match formatting of SQL Server INFORMATION_SCHEMA DATA_TYPE column value.
+    /// Sourced directly from internal/private SmiMetaData.cs in Microsoft.Data.SqlClient
+    /// Value indicates whether DAB engine supports the SqlDbType.
+    /// </summary>
+    /// <seealso cref="https://github.com/dotnet/SqlClient/blob/2b31810ce69b88d707450e2059ee8fbde63f774f/src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SmiMetaData.cs#L637-L674"/>
+    public static readonly Dictionary<string, bool> SupportedSqlDbTypes = new()
+    {
+            { "bigint", true },               // SqlDbType.BigInt
```

Comment:
the indentation seems off, looks like an extra tab from line#21 to line#51

Reply:
thanks, fixed!